x/ecocredit/client/tx.go

```diff
@@ -48,29 +48,40 @@ func txflags(cmd *cobra.Command) *cobra.Command {
 
 func TxCreateClassCmd() *cobra.Command {
 	return txflags(&cobra.Command{
-		Use:   "create-class [designer] [issuer[,issuer]*] [credit type] [metadata]",
-		Short: "Creates a new credit class",
-		Long: `Creates a new credit class.
+		Use:   "create-class [issuer[,issuer]*] [credit type] [metadata]",
+		Short: "Creates a new credit class with transaction author (--from) as admin",
+		Long: `Creates a new credit class with transaction author (--from) as admin.
+The transaction author must pay the fee associated with creating a new credit class.
```

Comment:
It might be nice to add a line to the long description about the transaction author needing  to be on the list of approved credit class creators if the allowlist is enabled.